### PR TITLE
Build push

### PR DESCRIPTION
**Pull request by zhilingc**
For #55.

Adds trigger to google cloud builds watching for semver tag pushes. The images are then pushed to gcr.

### CODE REVIEW

**Comment by tims** [comment]
actually if we are going to use cloud build for this, I don't think it should skip the tests.

/lgtm cancel

**Comment by zhilingc** [comment]
> actually if we are going to use cloud build for this, I don't think it should skip the tests.
> 
> /lgtm cancel

Okay, fair point. My rationale was that the presubmit tests would cover our bases, but better safe than sorry...

File: docker/core/Dockerfile [hunk 1]
```diff
@@ -1,9 +1,10 @@
 FROM openjdk:8-slim
```
Comment (woop) [inline docker/core/Dockerfile]: Should we bump this to Java 11?

https://jaxenter.com/nobody-puts-java-container-139373.html
https://bugs.openjdk.java.net/browse/JDK-8146115

Comment (zhilingc) [reply docker/core/Dockerfile]: Not sure if there are any compatibility issues, will test. 
Java 8 is the most ubiquitous so I'm not sure whether we should bump it or not if 11 is not backwards compatible

Comment (zhilingc) [reply docker/core/Dockerfile]: Performance is ok with the additional flags

Comment (woop) [reply docker/core/Dockerfile]: Ok, let's make it a new issue if necessary

Comment (tims) [reply docker/core/Dockerfile]: Dataflow currently does not work if you compile it with java 11. Java 8 is recommended

**Pull request merged**